core/runtime/jsrealm.rs

```diff
@@ -56,6 +58,8 @@ pub(crate) struct ContextState {
   pub(crate) wasm_instantiate_fn: RefCell<Option<Rc<v8::Global<v8::Function>>>>,
   pub(crate) unrefed_ops:
     RefCell<HashSet<i32, BuildHasherDefault<IdentityHasher>>>,
+  pub(crate) opcall_tracing_enabled: Cell<bool>,
+  pub(crate) opcall_traces: RefCell<HashMap<i32, String>>,
```

Comment:
In the future for perf, maybe it should be extracted out into a `struct FastValueCloneMapCell<T, U: Clone>(UnsafeCell<HashMap<T, U>>)` (or some other name) that clones the value on get in order to reduce the RefCell overhead.